swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/converter/ConverterMgr.java

```diff
@@ -124,6 +125,8 @@ private static void initPropertyMap() {
     PROPERTY_MAP.put(DateTimeProperty.class, SimpleType.constructUnsafe(Date.class));
 
     PROPERTY_MAP.put(ByteArrayProperty.class, SimpleType.constructUnsafe(byte[].class));
+
+    PROPERTY_MAP.put(FileProperty.class, SimpleType.constructUnsafe(String[].class));
```

Comment:
it's better map to Part?

Reply:
that means we have to construct part on the consumer side, which is very troublesome